pixi_diff_to_markdown/models.py

```diff
@@ -67,6 +67,9 @@ def transform_before(cls, data: Any) -> Any:
             )
 
         if "pypi" in data:
+            if data["pypi"].startswith("git+https://"):
```

Comment:
Could you add your minimal example from https://github.com/pavelzw/pixi-diff-to-markdown/issues/56#issuecomment-2761658977 to https://github.com/pavelzw/pixi-diff-to-markdown/blob/main/tests/resources/diff-example-v6-lockfile.json s.t. it's covered by the unit tests?

Reply:
done